test-cases/gutenberg/buttons.md

```diff
@@ -247,18 +247,7 @@
 
 ##### TC019
 
-### Fallback to default colors in case theme colors are used
-
--   Open gutenberg web, add `Buttons` block
--   Choose a background color from the **palette**
--   Choose a text color from the **palette**
```

Comment:
I'm wondering if we should keep this case around or not. I think if we do we'd want to have the app in bad state by setting the colors from one theme and then switch to another. Then make sure that the fall back works. 

WDYT?

Reply:
nope, we don't need to keep this but could you also update https://github.com/wordpress-mobile/test-cases/blob/master/test-suites/gutenberg/sanity-tests.md ?